webhooks/lambda/sandgrain/main.go

```diff
@@ -0,0 +1,69 @@
+// Due to go plugin mechanism,
+// the package of function handler must be main package
+package main
```

Comment:
This code should be out of the main repo. We have to place it in a different repo.